src/igvfd/tests/test_schema_sequence_file.py

```diff
@@ -380,3 +380,14 @@ def test_controlled_sequence_file_release(testapp, controlled_sequence_file_obje
         expect_errors=True
     )
     assert res.status_code == 200
+
+
+def test_analysis_step_version(testapp, sequence_file, analysis_step_version):
```

Comment:
Somewhat confused what this test is testing for -- just making sure that the property can be added to a file? If so, I think usually the inserts already cover making sure that the property can be used so we might not need this test.

Reply:
ok, removed